docs/dev/table/functions.md

```diff
@@ -5800,3 +5800,284 @@ For Table API, please use `_` for spaces (e.g., `DAY_TO_HOUR`).
 |                          | `SQL_TSI_SECOND ` _(SQL-only)_ |
 
 {% top %}
+
+Column Operations
```

Comment:
Column Functions ?

Reply:
I am also hesitating, because all of our functions can also be called expression, such as `=`, `<`, `>` These are Expressions, but our documentation is still called Comparison Functions, so I feel Expression is macro, and the functions are relatively specific. What's your opinion?

Reply:
Both Functions and Expressions make sense here. The columns() method is an Expression and also a Function. However, if we decided to put this method in `Built-In Functions`, naming it column Functions may be more consistent with the title? @dawidwys What do you think?

Reply:
> Could we convert this method to another Visitor? 

@dawidwys Considering your this comment. I think we can rename the `ColumnsOperationExpander` to `ColumnsFunctionExpander` and add a `ColumnsExpressionExpander`?